templates/ncrc5-cce.mk

```diff
@@ -90,7 +90,7 @@ FPPFLAGS := $(INCLUDES)
 FPPFLAGS += $(shell nf-config --fflags)
 
 # Base set of Fortran compiler flags
-FFLAGS = -s $(REAL_PRECISION) -s integer32 -h byteswapio -e m -h keepfiles -e0 -ez -N1023
+FFLAGS = -Wl,-rpath,${NETCDF_DIR}/lib:${HDF5_DIR}/lib -s $(REAL_PRECISION) -s integer32 -h byteswapio -e m -h keepfiles -e0 -ez -N1023
```

Comment:
This should not be in the `FFLAGS` or `CFLAGS`, but added to `LDFLAGS` below.

Reply:
okay thanks - made the change